boards/arm/circuitdojo_feather_nrf9160/board.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Circuit Dojo LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
+
+#define GPIO0 DT_LABEL(DT_NODELABEL(gpio0))
+#define POWER_LATCH_PIN 31
+
+static int board_circuitdojo_feather_nrf9160_init(struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	/* Get the device binding */
+	struct device *gpio = device_get_binding(DT_LABEL(DT_NODELABEL(gpio0)));
+
+	if (gpio == NULL) {
+		return -1;
```

Comment:
You should rather use some errno code here. `-ENODEV` seems suitable.

Reply:
Fixed!